agentops/instrumentation/anthropic/attributes/tools.py

```diff
@@ -0,0 +1,233 @@
+            for j, tool_use in enumerate(tool_uses):
+                tool_name = tool_use.get("name", "unknown")
+                tool_id = tool_use.get("id", f"tool-call-{j}")
+                
+                attributes[MessageAttributes.COMPLETION_TOOL_CALL_ID.format(i=0, j=j)] = tool_id
+                attributes[MessageAttributes.COMPLETION_TOOL_CALL_NAME.format(i=0, j=j)] = tool_name
+                attributes[MessageAttributes.COMPLETION_TOOL_CALL_TYPE.format(i=0, j=j)] = "function"
+                
+                tool_input = tool_use.get("input", {})
+                if isinstance(tool_input, dict):
+                    input_str = json.dumps(tool_input)
+                else:
+                    input_str = str(tool_input)
+                attributes[MessageAttributes.COMPLETION_TOOL_CALL_ARGUMENTS.format(i=0, j=j)] = input_str
+                
+                attributes[MessageAttributes.TOOL_CALL_ID.format(i=j)] = tool_id
+                attributes[MessageAttributes.TOOL_CALL_NAME.format(i=j)] = tool_name
+                attributes[MessageAttributes.TOOL_CALL_ARGUMENTS.format(i=j)] = input_str
+                attributes[f"{ToolAttributes.TOOL_STATUS}.{j}"] = ToolStatus.EXECUTING.value
+            
+            attributes["anthropic.tool_calls.count"] = len(tool_uses)
+        
+        tool_results = extract_tool_results(message_content)
+        if tool_results:
+            attributes["anthropic.tool_results"] = json.dumps(tool_results)
+            attributes["anthropic.tool_results.count"] = len(tool_results)
+            
+            for j, tool_result in enumerate(tool_results):
+                tool_use_id = tool_result.get("tool_use_id", "unknown")
+                
+                tool_index = None
+                for k in range(attributes.get("anthropic.tool_calls.count", 0)):
+                    if attributes.get(MessageAttributes.COMPLETION_TOOL_CALL_ID.format(i=0, j=k)) == tool_use_id:
+                        tool_index = k
+                        break
+                
+                if tool_index is not None:
+                    attributes[MessageAttributes.COMPLETION_TOOL_CALL_STATUS.format(i=0, j=tool_index)] = "complete"
+                    
+                    content = tool_result.get("content", {})
+                    if isinstance(content, dict):
+                        content_str = json.dumps(content)
+                    else:
+                        content_str = str(content)
+                    
+                    attributes[f"{ToolAttributes.TOOL_STATUS}.{tool_index}"] = ToolStatus.SUCCEEDED.value
+                    attributes[f"{ToolAttributes.TOOL_RESULT}.{tool_index}"] = content_str
+                    
+                    attributes[f"anthropic.tool_result.{tool_index}.content"] = content_str
+        
+    except Exception as e:
+        logger.debug(f"[agentops.instrumentation.anthropic] Error extracting tool attributes: {e}")
+    
+    return attributes 
```

Comment:
this file is low on test coverage. it should be pretty easy to add tests since this is all so deterministic. just make sure that the tests have correct anthropic schema

pls add tests :)